tools/cubemaptooctahedralmap/CopyOutputToSSBO.qml

```diff
@@ -1,3 +1,31 @@
+/*
+    CopyOutputToSSBO.qml
+
+    This file is part of Kuesa.
+
+    Copyright (C) 2018 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
```

Comment:
shouldn't it be 2019 ? (and is there a script that updates them all somewhere?)

Reply:
file was created in 2018 I think